.travis.yml

```diff
@@ -36,10 +36,10 @@ script:
 
 before_deploy:
   - curl -LSs https://box-project.github.io/box2/installer.php | php
-  - chmod +x box.phar
```

Comment:
I think the installer does it automatically when piped to `php`, when I downloaded it and ran through the steps yesterday, `box.phar` was already executable. I could add it back in just incase box-project changes that in the future though?